packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -131,18 +136,19 @@ export class ColumnHeaderCell extends React.Component<IColumnHeaderCellProps, IC
     };
 
     public render() {
-        const { className, isActive, isColumnSelected, loading, resizeHandle, style } = this.props;
+        const { loading } = this.props;
```

Comment:
> Also, it's not like this PR is small, so adding a few extra lines in nbd.

👏

> Also maybe that'll help you avoid lowering the test coverage threshold.

👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏

Regarding the first point, that's just the pattern I've noticed elsewhere in the Blueprint codebase. I don't really mind one way or the other. Seems like the destructuring approach makes sense here though.